animations is not optional in types and is missing from documentation

Since this commit, I have the following error in my project (when I try to upgrade this library):
> error TS2741: Property 'animations' is missing in type '{ annotations: { [key: string]: AnnotationOptions<keyof AnnotationTypeRegistry>; }; }' but required in type 'AnnotationPluginOptions'.

Shouldn't the `animations` property be optional ?
```diff
- animations: Record<string, unknown>,
+ animations?: Record<string, unknown>,
```

I don't find any documentation on this property in https://www.chartjs.org/chartjs-plugin-annotation/latest/ ; am I missing something ?

Thanks!

_Originally posted by @jbelien in https://github.com/chartjs/chartjs-plugin-annotation/pull/448#discussion_r768806334_

--- a/docs/guide/configuration.md
+++ b/docs/guide/configuration.md
@@ -11,6 +11,7 @@ The following options are available at the top level. They apply to all annotati
 | `clip` | `boolean` | No | `true` | Are the annotations clipped to the chartArea.
 | `drawTime` | `string` | Yes | `'afterDatasetsDraw'` | See [drawTime](options#draw-time)
 | `dblClickSpeed` | `number` | Yes | `350` | Time to detect a double click in ms.
+| [`animations`](#animations) | `object` | No | [see here](#default-animations) | To configure which element properties are animated and how
 
 :::warning
 
@@ -20,7 +21,43 @@ Nevertheless events are only catched over the chartArea.
 
 :::
 
-### Events
+## Animations
+
+Animations options configures which element properties are animated and how, with the same configuration of [chart.js](https://www.chartjs.org/docs/latest/configuration/animations.html#animations-2). 
+
+```javascript
+const options = {
+  plugins: {
+    annotation: {
+      animations: {
+        numbers: {
+          properties: ['x', 'y', 'x2', 'y2', 'width', 'height', 'radius'],
+          type: 'number'
+        },
+      },
+      annotations: {
+        box1: {
+          type: 'box',
+          xMin: 1,
+          xMax: 2,
+          yMin: 50,
+          yMax: 70,
+          backgroundColor: 'rgba(255, 99, 132, 0.5)'
+        }
+      }
+    }
+  }
+};
+```
+
+### Default animations
+
+| Name | Option | Value
+| ---- | ---- | ----
+| `numbers` | `properties` | `['x', 'y', 'x2', 'y2', 'width', 'height', 'pointX', 'pointY', 'labelX', 'labelY', 'labelWidth', 'labelHeight', 'radius']`
+| `numbers` | `type` | `number`
+
+## Events
 
 The following options are available for all annotation types. These options can be specified per annotation, or at the top level which apply to all annotations.
 

--- a/types/options.d.ts
+++ b/types/options.d.ts
@@ -133,5 +133,5 @@ export interface AnnotationPluginOptions extends AnnotationEvents {
   clip?: boolean,
   dblClickSpeed?: Scriptable<number, PartialEventContext>,
   drawTime?: Scriptable<DrawTime, PartialEventContext>,
-  animations: Record<string, unknown>,
+  animations?: Record<string, unknown>,
 }

